src/main/java/at/aau/serg/websocketdemoserver/deckmanagement/Deck.java

```diff
@@ -64,4 +64,22 @@ List<Card> getShuffledSubsetWithAddedGaia(int numOfCards) {
     int size() {
         return deck.size();
     }
+
+    public String evaluateTrick(LinkedHashMap<String, Card> playedCards) {
```

Comment:
suggestion evaluateWinningPlayerForRound to be more expressive
either way lets make sure that everyone is aware that the translation for trick is Stich